commands/challenge.js

```diff
@@ -0,0 +1,24 @@
+const { checkChallenge } = require('../utils/timeInterrupt');
+const { usersArray }  = require('./start');
+
+
+function ifChallenge(message){
+  if(message.author.bot) return;
+  if(message.content.startsWith('--challenge')){
```

Comment:
I think we said this would be automatically triggered when the user hits --exit in lockdown mode

Reply:
oh ignore me. This works as expected.